index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Diane Lenertz</title>
+  <link rel=StyleSheet href="styles/index-style.css">
+</head>
+<body>
+
+  <main>
+    <header class = "fade-in">
```

Comment:
In general I think we should try not to nest the major "page section" tags (`header`, `main`, `footer`). So we should try to have a page structure like:
```html
<body>
  <header>
    ...
  </header>
  <main>
    ...
  </main>
  <footer>
    ...
  </footer>
</body>
```

Reply:
Actually nevermind, I misunderstood the advice about using `header`, this use within `main` seems fine. Nesting the other way (`main` within `header`) would be an issue, however.